### PR TITLE
Add Richard Wild to website

### PR DESCRIPTION
**Pull request by richardjwild**

### CODE REVIEW

File: company/index.html [hunk 1]
```diff
@@ -54,11 +54,13 @@
                   </p>
                   {{member.fullDescription}}
                   <ul class="list-unstyled list-inline team-socail">
+                    {% if member.twitter != null %}
```
Comment (cyplo) [inline company/index.html]: May be better to have these in a separate PR, what do you think ? To not mix content changes with the behaviour changes

**Comment by cyplo** [comment]
Not sure about the photo - it's showing other people and the background is a bit distracting

File: company/index.html (hunk 1 above)
Comment (richardjwild) [reply company/index.html]: Okay, I'll put this change in a separate PR and we'll need to merge that one first.

Comment (cyplo) [reply company/index.html]: cool, you can say so in the description and reference them both using #763 notation

**Comment by richardjwild** [comment]
Fair comment on the photo. I've added a cleaner one and I think it fits with the other photos much better.

**Comment by richardjwild** [comment]
Please no-one merge this PR until #764 is merged first.
